app/controllers/authentication_controller.rb

```diff
@@ -2,17 +2,15 @@ class AuthenticationController < ApplicationController
   def token_auth
     token = params[:auth_token]
     payload = GoogleService.decode(token)
-    unless payload[:error].present?
-      if user = User.find_by(email: payload[:email])
-        user.google_token = token
-        user.save
-        render json: { is_new: false, token: token }
-      else
-        User.create!(name: payload[:name], email: payload[:email], google_token: token)
-        render json: { is_new: true, token: token }
-      end
-    else
+    if payload[:error].present?
       render json: payload, status: :bad_request
+    elsif user = User.find_by(email: payload[:email])
+      user.google_token = token
+      user.save
+      render json: { is_new: false, token: token }
+    else
+      User.create!(name: payload[:name], email: payload[:email], google_token: token)
+      render json: { is_new: true, token: token }
```

Comment:
FYI this was a rubocop fix -- we should all theoretically be running rubocop, so this one confused me a little. Seems to be identical anyway, but just in case you're surprised to find this here.

Reply:
maybe rubocop doesn't like `unless`